diff --git a/afirma-core/src/main/java/es/gob/afirma/core/misc/AOUtil.java b/afirma-core/src/main/java/es/gob/afirma/core/misc/AOUtil.java
index b0d25e2e9..91a57ae18 100644
--- a/afirma-core/src/main/java/es/gob/afirma/core/misc/AOUtil.java
+++ b/afirma-core/src/main/java/es/gob/afirma/core/misc/AOUtil.java
@@ -495,7 +495,7 @@ public final class AOUtil {
 			return false;
 		}
 		try {
-			if (Integer.parseInt(ver.substring(0, 1)) > 8) {
+			if (isOnlyNumber(ver) && Integer.parseInt(ver) > 8) {
 				return true;
 			}
 			// En el nuevo esquema de versionado de Java se sigue el patron [1-9][0-9]*((\.0)*\.[1-9][0-9]*)*,
