diff --git a/afirma-keystores-mozilla/src/main/java/es/gob/afirma/keystores/mozilla/MozillaKeyStoreUtilitiesUnix.java b/afirma-keystores-mozilla/src/main/java/es/gob/afirma/keystores/mozilla/MozillaKeyStoreUtilitiesUnix.java
index 0342d2423..c223c24d3 100644
--- a/afirma-keystores-mozilla/src/main/java/es/gob/afirma/keystores/mozilla/MozillaKeyStoreUtilitiesUnix.java
+++ b/afirma-keystores-mozilla/src/main/java/es/gob/afirma/keystores/mozilla/MozillaKeyStoreUtilitiesUnix.java
@@ -29,7 +29,10 @@ final class MozillaKeyStoreUtilitiesUnix {
 
 	private static final String SOFTOKN3_SO = "libsoftokn3.so"; //$NON-NLS-1$
 
-	private static final String[] NSS_PATHS = getNssPaths();
+	private static final String[] NSS_PATHS = {
+		"@nsspath",
+		"@firefoxpath"
+	};
 
 	private static final String[] SQLITE_LIBS = {
 		"mozsqlite3.so",  //$NON-NLS-1$
